plat/xilinx/zynqmp/include/platform_def.h

```diff
@@ -34,9 +34,14 @@
  * little space for growth.
  */
 #ifndef ZYNQMP_ATF_MEM_BASE
+#if !defined(DEBUG)
```

Comment:
Can you check that this is working as you expect? I believe DEBUG is always defined - perhaps should be #if DEBUG?

Reply:
Hi,
Yes, it doesn't work, we will be correct it with if !DEBUG instead.